example-conf/mesosphere-dcos/hdfs-site.xml

```diff
@@ -0,0 +1,207 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<?xml-stylesheet type="text/xsl" href="configuration.xsl"?>
+<!--
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License. See accompanying LICENSE file.
+-->
+
+<!-- Put site-specific property overrides in this file. -->
+
+<configuration>
+  <property>
+    <name>dfs.ha.automatic-failover.enabled</name>
+    <value>true</value>
+  </property>
+
+  <property>
+    <name>dfs.nameservice.id</name>
+    <value>hdfs</value>
+  </property>
+
+  <property>
+    <name>dfs.nameservices</name>
+    <value>hdfs</value>
+  </property>
+
+  <property>
+    <name>dfs.ha.namenodes.hdfs</name>
+    <value>nn1,nn2</value>
+  </property>
+
+  <property>
+    <name>dfs.namenode.rpc-address.hdfs.nn1</name>
+    <value>namenode1.hdfs.mesos:50071</value>
+  </property>
+
+  <property>
+    <name>dfs.namenode.http-address.hdfs.nn1</name>
+    <value>namenode1.hdfs.mesos:50070</value>
+  </property>
+
+  <property>
+    <name>dfs.namenode.rpc-address.hdfs.nn2</name>
+    <value>namenode2.hdfs.mesos:50071</value>
+  </property>
+
+  <property>
+    <name>dfs.namenode.http-address.hdfs.nn2</name>
+    <value>namenode2.hdfs.mesos:50070</value>
+  </property>
+
+  <property>
+    <name>dfs.ha.automatic-failover.enabled.hdfs</name>
+    <value>true</value>
+  </property>
+
+  <property>
+    <name>dfs.client.failover.proxy.provider.hdfs</name>
+    <value>org.apache.hadoop.hdfs.server.namenode.ha.ConfiguredFailoverProxyProvider</value>
+  </property>
+
+  <property>
+    <name>dfs.namenode.shared.edits.dir</name>
+    <value>qjournal://journalnode1.hdfs.mesos:8485;journalnode2.hdfs.mesos:8485;journalnode3.hdfs.mesos:8485/hdfs</value>
+  </property>
+
+  <property>
+    <name>ha.zookeeper.quorum</name>
+    <value>master0.mesos:2181</value>
+  </property>
+
+  <property>
+    <name>dfs.journalnode.edits.dir</name>
+    <value>/tmp/hdfs/data/jn</value>
+  </property>
+
+  <property>
+    <name>dfs.namenode.name.dir</name>
+    <value>file:///tmp/hdfs/data/name</value>
```

Comment:
Can we come up with a non-/tmp default location that will still work on DCOS? No problem if we can't, since it is configurable if a customer has their own setup.